packages/wrangler/src/proxy.ts

```diff
@@ -328,6 +329,10 @@ export function usePreviewServer({
 			.then(() => {
 				proxy.server.on("listening", () => {
 					logger.log(`⬣ Listening at ${localProtocol}://${ip}:${port}`);
```

Comment:
Is there a reason we don't list them all even in the scenario of someone setting a specific one?

Reply:
If someone sets a specific one, it's very possible the others will not be being listened on, unlike 0.0.0.0 which listens on all addresses